ios/RCTConvert+RNCarPlay.m

```diff
@@ -51,16 +51,19 @@ + (CPMapButton*)CPMapButton:(id)json withHandler:(void (^)(CPMapButton * _Nonnul
 
 + (MKMapItem*)MKMapItem:(id)json {
     CLLocationCoordinate2D coordinate = CLLocationCoordinate2DMake([RCTConvert double:json[@"latitude"]], [RCTConvert double:json[@"longitude"]]);
+    NSLog(@">>> Parsing mapItem %@", json);
```

Comment:
Maybe have the nslog under a debug flag unless there is a reason for it

Reply:
I can drop the NSLogs, these are likely left over from my debugging (I kept losing references to the map items due to the trips being recreated every time 😢 )

Reply:
@birkir NSLog calls removed